test/src/unit-capi-config.cc

```diff
@@ -226,6 +226,7 @@ void check_save_to_file() {
   ss << "config.logging_level 0\n";
 #endif
   ss << "filestore.buffer_size 104857600\n";
+  ss << "rest.capnp_traversal_limit 512\n";
```

Comment:
Should this be in  bytes?

Reply:
@Shelnutt2 I wanted to make it bytes, but I thought it's going to be annoying when tweaking to get an outlier use case across cause you'll be trying with big values > 512mb.
I can change if you want to, MBs seemed like a better idea to me.

Reply:
I agree with you on usability, however other configs are bytes so I think consistency might be the better approach.

Reply:
bytes it is then.

Reply:
Thanks!